components/feature/sitepermissions/src/main/java/mozilla/components/feature/sitepermissions/SitePermissionsFeature.kt

```diff
@@ -551,3 +556,16 @@ private fun isPermissionGranted(
             throw InvalidParameterException("$permission is not a valid permission.")
     }
 }
+
+private fun Permission.isSupported(): Boolean {
+    return when (this) {
+        is ContentGeoLocation,
+        is ContentNotification,
+        is ContentAudioCapture, is ContentAudioMicrophone,
+        is ContentVideoCamera, is ContentVideoCapture -> {
+            true
```

Comment:
nit: should be `-> true` and `-> false` or consistent with each other some other way

Reply:
👍

Reply:
Updated!